src/modules/repository/implementation/sequelize/sequelize-repository.js

```diff
@@ -502,10 +501,11 @@ class SequelizeRepository {
         return inserted.map((event) => event.dataValues);
     }
 
-    async getAllUnprocessedBlockchainEvents() {
+    async getAllUnprocessedBlockchainEvents(eventNames) {
```

Comment:
we need to add a migration to remove old unprocessed events, just to make sure that on start, the node won't process old unprocessed events